runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4281,11 +4273,7 @@ done:;
 		walkState->userData1 = (void*)(UDATA)(depth + 1); /* Skip the local INL frame */
 		walkState->userData2 = NULL;
 		walkState->userData3 = (void *) FALSE;
-		walkState->frameWalkFunction = cInterpGetStackClassIterator;
-
-		if ((J2SE_VERSION(_vm) & J2SE_VERSION_MASK) >= J2SE_18) {
-			walkState->frameWalkFunction = cInterpGetStackClassJEP176Iterator;
-		}
+		walkState->frameWalkFunction = cInterpGetStackClassJEP176Iterator;
```

Comment:
Seems the function cInterpGetStackClassIterator can be deleted now.

Reply:
Deleted. Pls have another look.